cmd/add.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"github.com/matsuyoshi30/gitsu/cmd/prompts"
+	"github.com/matsuyoshi30/gitsu/internal/config"
+	"github.com/matsuyoshi30/gitsu/internal/models"
+
+	"github.com/urfave/cli/v2"
+)
+
+// AddCommand returns the definition for the 'gitsu add' command
+func AddCommand() *cli.Command {
+	return &cli.Command{
+		Name:    "add",
+		Aliases: []string{"a"},
+		Usage:   "Add new user",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "gpg",
+				Value: false,
+				Usage: "Add GPG key ID",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			name, err := prompts.Input("Git user name")
+			if err != nil {
+				return err
+			}
+
+			email, err := prompts.InputWithValidation(
+				"Git user email",
+				func(s string) error {
+					return models.ValidateEmail(s, false)
+				},
+			)
+			if err != nil {
+				return err
+			}
+
+			var keyID string
+			if c.Bool("gpg") {
+				keyID, err = prompts.Input("GPG key ID")
+				if err != nil {
+					return err
+				}
+			}
+
+			alias, err := prompts.Input("User alias, leave empty for no alias")
+			if err != nil {
+				return err
+			}
+
+			user := models.NewUser(name, email, alias, keyID)
+			cfg, err := config.Read()
```

Comment:
Those who use it for the first time cannot add a git user because `config.Read()` returns an error (config file doesn't exist).

I think it has to check config file existence before `config.Read()` and create empty config file if it doesn't exist.

Reply:
This is now fixed by commit 01df57b. Let me know if everything works as expected!